src/adapters/DebugProtocolAdapter.ts

```diff
@@ -636,7 +638,7 @@ export class DebugProtocolAdapter {
      * @param name the name of this variable. For example, `alpha.beta.charlie`, this value would be `charlie`. For local vars, this is the root variable name (i.e. `alpha`)
      * @param parentEvaluateName the string used to derive the parent, _excluding_ this variable's name (i.e. `alpha.beta` or `alpha[0]`)
      */
-    private createEvaluateContainer(variable: Variable, name: string, parentEvaluateName: string) {
+    private createEvaluateContainer(variable: Variable, name: string | number, parentEvaluateName: string) {
```

Comment:
Can you add a few unit tests around passing in a name as string, then a name as number, and make sure the output evaluateName correctly handles the `["key"]` or `[123]` style?

Reply:
Pushed up two tests. Let me know if you have any issues with it.